demo/static/sass/demo-ltr.scss

```diff
@@ -9,6 +9,11 @@
 // ------------------------------
 $layout-direction: ltr;
 
+// ------------------------------
+// #LIB
+// ------------------------------
+@import '../../../node_modules/bi-app-sass/bi-app/bi-app-ltr';
```

Comment:
Nit: I'm wondering if we should add partials for _ltr.scss and _rtl.scss. I'm not super comfortable with every client of the pattern library having to know that we're using bi-app, and it would make it harder to change that dependency. I'm also wondering if we might eventually have some additional actions that we'd perform in these partials, so having them broken out now would make that easier.

Does that make sense? Am I missing something as to why this is a bad idea?